app/ui/shared/widgets/list-item.jsx

```diff
@@ -22,7 +24,10 @@ const LIST_ITEM_STYLE = Style.registerStyle({
 
 class ListItem extends Component {
   shouldComponentUpdate(nextProps) {
-    return !isEqual(this.props, nextProps);
+    // Children in this component may contain immutable.js props, which are
```

Comment:
The warnings are one thing, but also this is changing the behavior of shouldComponentUpdate to return false when child components re-render, and I think that's important to capture in this comment.